Solutions/Azure Active Directory/Analytic Rules/AccountCreatedDeletedByNonApprovedUser.yaml

```diff
@@ -26,18 +26,18 @@ query: |
   AuditLogs
   | where OperationName == "Add user" or OperationName == "Delete user"
```

Comment:
Whilst we are improving this we could also make some improvements such as making == to =~ to make  it more robust